src/containers/Cluster/Cluster.tsx

```diff
@@ -108,17 +115,20 @@ export function Cluster({
     }, [dispatch]);
 
     const actualClusterTabs = React.useMemo(() => {
-        let tabs = clusterTabs;
+        const skippedTabs: ClusterTab[] = [];
 
         if (!shouldShowNetworkTable) {
-            tabs = tabs.filter((tab) => tab.id !== clusterTabsIds.network);
+            skippedTabs.push(clusterTabsIds.network);
         }
         if (!shouldShowEventsTab) {
-            tabs = tabs.filter((tab) => tab.id !== clusterTabsIds.events);
+            skippedTabs.push(clusterTabsIds.events);
+        }
+        if (!isViewerUser) {
```

Comment:
You should check also here. If no configs there still will be a tab in tablist

Reply:
Omg, sorry :) I slept very little last night